### DISCUSSION TITLE
Add New issues

### DISCUSSION THREAD

**Discussion by Studio88-KM** [code4recovery/12-step-meeting-list#950] [Q&A] [answered]
Hi there, I'm testing the Add New meeting. 

When I click "Add New" and get to the CPT screen, I see the Publish button. However, if I add incorrect data (I believe) into the URL field for Zoom or wrong format for the Phone number / password field or if I choose Region first for entering an address, the Publish button goes away and gets greyed out and I can only save it as a draft. 

The problem is that when I remove the information, the Publish button doesn't come back. It stays greyed out. So, I have to start all over again with a new post. This is a pain to lose all the work I've done so far (and more importantly, I don't want others to have to do that). 

Maybe I'm missing something?

Also, what is the correct way to fill in an address? It seems that no matter what I add first or how I add it (format-wise), I get the greyed out Publish button. I tested it with my address knowing it's a real address and I got the greyed out button. That could be because I'm in a state that the website/site doesn't cover but please let me know the parameters. 

Can you advise/help and also let me know what the correct formats are for URL/Zoom, phone number, address?

Thanks in advance!

**Comment by Studio88-KM** [comment]
Actually, I don't think it's just when it's invalid data. I copied the Zoom link form a valid meeting showing on our site and after I pasted it into the the URL field of the test and the Publish button went away. What am I missing here?

**Comment by joshreisner** [comment, marked as answer]
hm, sorry to hear you're having trouble. sounds like there might be a geocoding issue? If you go to `Meetings` > `Import & Settings` -> `Settings` -> `Address Geocoding` - what's that set to? You might try changing the value of that to see if it helps.

If not, could you attach a screenshot or tell us what values you're entering? We can give it a try on our demo site

**Reply by Studio88-KM** [reply]
Thanks for getting back! Sorry, I didn't get an email indicating a response! 

It's set to the Beta API Gateway which is the look we're choosing. Should we try the Legacy since we're not choosing that look? 

By values, do you mean what are we directly typing into the fields or the values in Settings? 

Yes, I'd love for you to try it on your demo.

**Reply by joshreisner** [reply]
Try changing to legacy geocoding, that won't affect the appearance of the site.

if that doesn't work, please add a screenshot with the physical address and zoom address that you're using.

**Reply by Studio88-KM** [reply]
Thanks. Yeah, that didn't work. I'll send a Loom video as soon as I can.

**Reply by Studio88-KM** [reply]
Thanks for your patience with this. Here's a loom video: https://www.loom.com/share/3e3e699f419f498ea00cee0189e07c8b

**Reply by tim-rohrer** [reply]
The recording plays on my iPad for 6 seconds, then stops and goes black.

Hopefully someone else will have better luck.

**Reply by joshreisner** [reply]
Thanks @Studio88-KM - the video works for me

I think this may be an issue of a confusing UI more than anything. I think you need to keep going and add location information to your meeting - in the case of an online meeting you will select that it cannot be attended in person, and you will be advised to enter a general location, such as `New York, NY, USA` - then the button will be re-enabled.

I think the UI should probably do a better job of educating users that meetings must have location info entered to be able to move forward, rather than simply greying out the button with no explanation.

This design pattern came about initially during the days before online meeting info could be entered, and it would happen if someone entered a location that wasn't findable, eg "Ted's House" instead of a valid street address.

**Reply by Studio88-KM** [reply]
My bad. I totally missed reading that small text. Maybe have a Warning or Important above it. 

![image](https://user-images.githubusercontent.com/77176347/195406797-ac97009f-84fd-4aac-90c5-5f22d1717738.png)

Thanks so much for pointing it out! And thanks so much for your help!
